### DISCUSSION TITLE
Icon sets

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1248] [Q&A] [answered]
How can I add an additional icon set?

**Comment by meetmandeep** [comment, marked as answer]
Create a new folder at Root\Icons\FolderName and add your SVG files.

**Reply by scottw12** [reply]
So I have to handle fontawesome as SVG's rather than through a CDN...in order to have them included in the icon block.

OK

**Reply by meetmandeep** [reply]
Yes. You can always include CSS via theme builder or Custom Styling if you want a font base / CDN delivered font set. 

For a native integration you'll need SVGs which were chosen over Font Based CDN for performance (site speed) and ability to infinitely scale / resize without and degradation or performance impact.

**Reply by scottw12** [reply]
Understood, remembered that it was there in the first beta. Now I understand the thoughts behind and makes sense to me.

**Reply by Tychodewaard** [reply]
May I just add that this really simple and awesome! 
I have seen many CMSses and many implementations where a designer comes up with custom icons and you know it's going to be a lot of work. But this is how it should be :-)
